acctest/auth.py

```diff
@@ -0,0 +1,53 @@
+# /// script
```

Comment:
This is a bit of a hack. To run these tests manually, the user would first run `oxide auth login`, but that's an interactive flow that requires a browser. To make this non-interactive, we add a script that simulates the device auth flow and prints a token. @david-crespo pointed out that we could instead support password login via the cli, with a warning not to use it in production, but IMO this is fine for the time being.